### PR TITLE
fix: mobile, tablet view

### PR DESCRIPTION
**Pull request by ocmodi21**
mobile-view-updated
ipad-view-updated-1
laptop-view

### CODE REVIEW

**Comment by SujithThirumalaisamy** [comment]
Can you please resolve the conflicts

**Comment by ocmodi21** [comment]
@SujithThirumalaisamy 

I resolved conflicts, Please review it.

**Comment by SujithThirumalaisamy** [comment]
Can you please run lint fix and push the changes

File: next-app/components/Appbar.tsx [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
-        {!session.data?.user && (
-          <Button
-            className="bg-purple-600 text-white hover:bg-purple-700"
-            onClick={() => signIn()}
-          >
-            Signin
-          </Button>
-        )}
-      </div>
+    <div className="flex justify-between px-5 md:px-10 xl:px-20 py-4">
```
Comment (SujithThirumalaisamy) [inline next-app/components/Appbar.tsx:13]: These are breaking the lints again

Comment (ocmodi21) [reply next-app/components/Appbar.tsx:13]: {E9888673-6BAF-4D3E-92C7-BE68E28A3AC8}

{E995039F-E0CD-4D0B-9F17-D7B44319E6A2}


When I perform the lint fix, no files have changed.

Comment (SujithThirumalaisamy) [reply next-app/components/Appbar.tsx:13]: `pnpm format:fix`

Comment (ocmodi21) [reply next-app/components/Appbar.tsx:13]: Done

**Pull request merged**